src/igvfd/schemas/file.json

```diff
@@ -33,6 +33,9 @@
         }
     ],
     "dependentSchemas": {
+        "$merge": [
+            "shared_dependencies.json#/"
```

Comment:
I think you need to merge specific shared dependencies, e.g.: 
https://github.com/IGVF-DACC/igvfd/blob/ff1fabc4bf285e161af3b5cdc9ed1be0cd972212/src/igvfd/schemas/degron_modification.json#L48-L49